test/cppsuite/test_harness/thread_context.h

```diff
@@ -82,9 +84,54 @@ class thread_context {
         _running = running;
     }
 
+    void
+    begin_transaction(WT_SESSION *session, const std::string &config)
+    {
+        if (!_in_txn) {
+            testutil_check(
+              session->begin_transaction(session, config.empty() ? NULL : config.c_str()));
+            /* This randomizes the number of operations to be executed in one transaction. */
+            _max_op_count = random_generator::instance().generate_integer(_min_op, _max_op);
+            _current_op_count = 0;
+            _in_txn = true;
+        }
+    }
+
+    /* Returns true if the current transaction has been committed. */
+    bool
+    commit_transaction(WT_SESSION *session, const std::string &config)
+    {
+        /* A transaction cannot be committed if not started. */
+        testutil_assert(_in_txn);
+        /* The current transaction should be committed if:
+         *  - The thread is done working. This is useful when the test is ended and the thread has
+         * not reached the maximum number of operations per transaction.
+         *  - The number of operations executed in the current transaction has exceeded the
+         * threshold.
+         */
+        if (!_running || (++_current_op_count > _max_op_count)) {
+            testutil_check(
+              session->commit_transaction(session, config.empty() ? NULL : config.c_str()));
+            _in_txn = false;
+        }
+
+        return !_in_txn;
```

Comment:
wrap in brackets.

Reply:
Will fix the other return statements in this class too